feat: add support for latest versions, plus VerifiableURI support (#55)
lukso-network/lsp-utils: docs/modules/LSP3ProfileMetadata.md

#### Defined in

[LSP3ProfileMetadata/isProfileMetadata/isProfileMetadata.ts:16](https://github.com/lukso-network/lsp-utils/blob/31b2f8b/src/LSP3ProfileMetadata/isProfileMetadata/isProfileMetadata.ts#L16)

---

### setProfileMetadata

▸ **setProfileMetadata**(`erc725y`, `json`, `url`, `signer?`): `Promise`\<`void`\>

Set Profile metadata.

#### Parameters

| Name      | Type                      | Description                                                                  |
| :-------- | :------------------------ | :--------------------------------------------------------------------------- |

> | `erc725y` | `BytesLike` \| `ERC725Y`  | The address of the Profile whoose metadata needs to be updated.              |

Reviewer: ```suggestion
| `erc725y` | `BytesLike` \| `ERC725Y`  | The address of the Profile to set the metadata for.              |
```

> | `json`    | `LSP3ProfileMetadataJSON` | The JSON that was hosted at `url`.                                           |

Reviewer: why talking at the past here?

```suggestion
| `json`    | `LSP3ProfileMetadataJSON` | The JSON file content hosted at `url`.                                           |
```

| `url`     | `string`                  | The URL where the JSON file is hosted.                                       |

> | `signer?` | `Signer` \| `Wallet`      | The etehrs `Signer`, address that is allowed ot modify the Profile metadata. |

Reviewer: ```suggestion
| `signer?` | `Signer` \| `Wallet`      | The ethers `Signer`, address that is allowed to modify the Profile metadata. |
```
